src/prx/rinex_nav/evaluate.py

```diff
@@ -9,11 +9,21 @@
 import georinex
 from prx import util
 from prx import constants
-from prx.util import timeit, repair_with_gfzrnx
+from prx.util import timeit, repair_with_gfzrnx, compute_gps_utc_leap_seconds
 
 log = logging.getLogger(__name__)
 
 
+def detect_leap_second_insertion(df: pd.DataFrame):
```

Comment:
Adds an additional check that asserts even when the nav file does not have leap seconds.